p2p/protocol/identify/id.go

```diff
@@ -352,14 +369,19 @@ func (ids *IDService) identifyConn(c network.Conn, signal chan struct{}) {
 		ids.removeConn(c)
 		return
 	}
-	s.SetProtocol(ID)
 
-	// ok give the response to our handler.
-	if err = msmux.SelectProtoOrFail(ID, s); err != nil {
+	// TODO This might cause us an extra roundtrip if the remote peer does not not support
```

Comment:
@Stebalien Either we "disable" `ID_1_1_0 ` for now(since we don't really need signed records anywhere) or we pay this cost. I don't there's any other clean solution till we have multistream2.0.

Lmk what you think.

Reply:
> (since we don't really need signed records anywhere) 

That's decisively not true. Both lotus and (especially) drand rely on signed peer records to bootstrap pubsub through Peer Exchange without a DHT.

Reply:
@vyzo I get what you mean. I'll make the configuration changed discussed in #954 .